Bottom: 8b5f658b741ce7cf477b4e20911363e3c864ff9c
Top:    97a2ac86ed1158643b870e6289a8a802a29b1071
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-13 01:20:05 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 11c75d2..98ab484 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
